lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -708,9 +708,10 @@ def add_processing_options(xml, options)
         return unless options[:stored_credential]
 
         xml.processingOptions do
-          if options[:stored_credential][:initial_transaction]
```

Comment:
They will be set to isFirstRecurringPayment, isFirstSubsequentAuth, or isStoredCredentials if the conditions are met. Otherwise, it will be a follow-on transaction for an established customer and isSubsequentAuth will be set.